scheduler/pkg/server/utils.go

```diff
@@ -0,0 +1,35 @@
+/*
+Copyright (c) 2024 Seldon Technologies Ltd.
+
+Use of this software is governed by
+(1) the license included in the LICENSE file or
+(2) if the license included in the LICENSE file is the Business Source License 1.1,
+the Change License after the Change Date as each is defined in accordance with the LICENSE file.
+*/
+
+package server
+
+import (
+	"time"
+
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+func sentWithTimeout(f func() error, d time.Duration) (bool, error) {
```

Comment:
I propose we rename this to `sendWithTimeout`

Reply:
done in [b90056a](https://github.com/SeldonIO/seldon-core/pull/5434/commits/b90056a15731f316d43d718cdaec388f1f3b34ee)